test/tc-yaml_test.js

```diff
@@ -26,6 +26,7 @@ suite('tc-yaml_test.js', function() {
       assume(config.tasks).to.deeply.equal([{
         taskId: config.tasks[0].taskId,
         task: {
+          created: new Date().toJSON(),
```

Comment:
This doesn't pass :(

> Unknown assertation failure occured, assumed `[ { task: { created: '2018-09-13T15:24:27.542Z', schedulerId: 'test-sched', taskGroupId: 'fxVScA_cRdejLPnJXtKFrQ' }, taskId: 'fxVScA_cRdejLPnJXtKFrQ' } ]` to deeply equal `[ { task: { created: '2018-09-13T15:24:27.544Z', schedulerId: 'test-sched', taskGroupId: 'fxVScA_cRdejLPnJXtKFrQ' }, taskId: 'fxVScA_cRdejLPnJXtKFrQ' } ]`

Note that the milliseconds on the timestamps are different.

Usually the best fix for this sort of problem is to pass in a single value for `now` that the test can specify and then compare.  Something like

```
const now = new Date();
tcyaml.compileTasks(config, cfg, {}, now);
assume(config.tasks).to.deeply.equal(... created: now.toJSON() ...);
```

I don't recall if we create a new TcYaml instance for every new Github event.  If we do, then it might make more sense to set `now` in the VersionOne constructor.  Then there's no need to pass `now` to `compileTasks`, and the tests can just access the `tcyaml.now` value for checks:

```
assume(config.tasks).to.deeply.equal(... created: tcyaml.now.toJSON() ...);
```

Reply:
Thank you Dustin!